src/services/character_creation.py

```
@@ -0,0 +1,75 @@
+                print("  Invalid input. Enter a number between 1-3.")
+            except KeyboardInterrupt:
+                return None
+
+    @staticmethod
+    def _ensure_class_art(char_class: CharacterClass) -> None:
+        """Ensure character class has associated art"""
+        if not hasattr(char_class, "art") or not char_class.art:
+            art_file = ensure_character_art(char_class.name)
+            if art_file:
+                art_content = load_ascii_art(art_file)
+                setattr(char_class, "art", art_content)
```

Ruff (0.8.0)

Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)